app/jobs/reports/drop_off_report.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'reporting/drop_off_report'
+
+module Reports
+  class DropOffReport < BaseReport
+    REPORT_NAME = 'drop-off-report'
+
+    attr_accessor :report_date
+
+    def perform(report_date, frequency = 'weekly')
```

Comment:
I think we need to schedule this job in the `job_configurations.rb`

Also it's not common for us to have optional arguments like `frequency` like we have here. Is this something that might be better suited as a configuration value so we can change it without a code deploy?